constants/util.go

```diff
@@ -69,10 +70,11 @@ var (
 			DHubURL:          "docker.io/bcgsc/abyss", // https://github.com/bcgsc/abyss
 			OutputDir:        AbyssOut,
 			AssemblyFileName: "final-contigs.fa",
-			Comm: func() []string {
+			Comm: func(threads int) []string {
 				return []string{
 					`k=25`,
 					`name=final`,
+					fmt.Sprintf("j=%s", string(threads)),
```

Comment:
Instead of turning it into a string, this can be `fmt.Sprintf("j=%d", threads)`.